fiftyone/core/session/session.py

```
@@ -1440,6 +1487,49 @@ def _resolve_selection_style(
+    labels: t.List,
+) -> t.List[t.Dict]:
+    normalized = []
+    for label in labels:
+        if isinstance(label, dict):
+            label.setdefault("type", "default")
+            sel_type = label["type"]
+            if sel_type not in VALID_SELECTION_TYPES:
+                raise ValueError(
+                    f"Invalid selection type '{sel_type}'. "
+                    f"Must be one of {VALID_SELECTION_TYPES}"
+                )
```

Ruff (0.15.9)

[warning] Avoid specifying long messages outside the exception class

(TRY003)